components/backup-stream/src/endpoint.rs

```diff
@@ -321,8 +321,6 @@ where
             if task.is_paused {
                 continue;
             }
-            // We have meet task upon store start, we must in a failover.
```

Comment:
@YuJuncen mentioned we don't need this anymore, the relevant unit test was removed also as part of the cleanup